salt/cloud/clouds/profitbricks.py

```diff
@@ -626,6 +626,40 @@ def list_nodes_full(conn=None, call=None):
     return ret
 
 
+def reserve_ipblock(call=None, kwargs=None):
+    '''
+    Reserve the IP Block
+    '''
+    if call == 'action':
+        raise SaltCloudSystemExit(
+            'The reserve_ipblock function must be called with -f or '
+            '--function.'
+        )
+
+    conn = get_conn()
+
+    if kwargs is None:
+        kwargs = {}
+
+    ret = {}
+    ret['ips'] = []
+
+    if kwargs.get('location') is None:
+        raise SaltCloudExecutionFailure('The "location" parameter is required')
+    location = kwargs.get('location')
+
+    size = 1
+    if kwargs.get('size') is not None:
+        size = kwargs.get('size')
+
+    block = conn.reserve_ipblock(IPBlock(size=size, location=location))
+    pprint.pprint(block)
```

Comment:
Can you remove this print statement? You can use logging if necessary, but we don't like to print directly this way.

Reply:
@rallytime The print statement removed.